unit/src/main/include/robot/outputs.h

```diff
@@ -22,6 +22,11 @@ class Outputs {
     // BlinkingLED
     std::string ledCommand;
 
+    // Camera Movement
+    std::vector<double> positionCamera;
+    std::vector<double> targetCameraPosition;
+    std::vector<double> previousCameraPosition;
```

Comment:
targetCameraPosition and previousCameraPosition in outputs.h are in the wrong place. (target should be in commands and previous in state?)